sqlglot/dialects/sqlite.py

```diff
@@ -27,6 +27,16 @@ def _date_add_sql(self: SQLite.Generator, expression: exp.DateAdd) -> str:
     modifier = f"'{modifier} {unit.name}'" if unit else f"'{modifier}'"
     return self.func("DATE", expression.this, modifier)
 
+def str_to_date_sql(self: SQLite.Generator, expression: exp.StrToDate) -> str:
+    date_string = self.sql(expression, "this")
+    format_specifier = self.format_time(expression)
+    return f"STRFTIME({format_specifier}, {date_string})"
+
+
+def str_to_time_sql(self: SQLite.Generator, expression: exp.StrToTime) -> str:
+    time_string = self.sql(expression, "this")
+    format_specifier = self.format_time(expression)
+    return f"STRFTIME({format_specifier}, {time_string})"
```

Comment:
These look identical, can we consolidate them into a single function and then reuse it for both `StrToDate` and `StrToTime` to avoid code duplication?